sdk/servicebus/service-bus/package.json

```
@@ -55,7 +55,7 @@
     "build:types": "downlevel-dts types/latest types/3.1",
     "build": "tsc -p . && rollup -c 2>&1 && npm run extract-api && npm run build:types",
     "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf dist* test-dist types *.tgz *.log coverage coverage-browser .nyc_output",
+    "clean": "rimraf dist* dist-test types *.tgz *.log coverage coverage-browser .nyc_output",
```

```suggestion
    "clean": "rimraf dist* types *.tgz *.log coverage coverage-browser .nyc_output",
```
dist* covers dist-test, right?

---

```suggestion
    "clean": "rimraf dist dist-* types *.tgz *.log coverage coverage-browser .nyc_output",
```